kedro/contrib/idempotent/idempotent_runner.py

```diff
@@ -100,7 +103,6 @@ class IdempotentSequentialRunner(AbstractRunner):
     def __init__(self):
         super().__init__()
         self.force_run = False
-        self.state_storage = IdempotentStateStorage()
```

Comment:
Add in `self.state_storage = None` so that the class is always correctly initialized.

Reply:
Think about forgoing the previous comment, depending on how you decide to instantiate this class.